### DISCUSSION TITLE
Mongo DB and MySQL?

### DISCUSSION THREAD

**Discussion by smokinjo** [socioboard/Socioboard-5.0#329] [Q&A]
While trying to install the open source version, our technician was was wondering why socioboard is using MOngo DB and MySQL?

Thanks

Joseph

**Comment by NickAtkinson** [comment]
Hello Joseph,

I realize your question is over a month old, but I have a follow-up question for you at the end.

I don't have an exact answer here, but it's not that uncommon to use both mongo and mySQL:

> It's common to use MySQL as the primary storage and MongoDB as caching/intermediate storage for speed.
> You can for example have read-intensive data in MongoDB. The data for generating reports is perfect for a relational system like MySQL.

I'm not _entirely_ convinced that's the case here, but that's my best guess.

Were you able to get Socioboard working?

-Nick

**Reply by smokinjo** [reply]
HI NIck,

Thanks for the reply.

We have made some progress, but the system is not working.  We've made progress by trying to fix errors as they show up.   WE still can not log in to the system.

I am not looking forward to seeing how it handles posting messages and monitoring them.  Hopefully this part is running better than getting it working to be able to log in.

Have you installed this yourself?

The developers suggest help in the forums, but after trying the forums, email, etc...  The developers have not been forthcoming with assistance or words of guidance.

Thanks

**Reply by NickAtkinson** [reply]
I'm in pretty much the same situation, but I have it at a point where I can log in. I was in the process of setting up Facebook, but my free trial expired on my test user, so I tried to get that user set up as an admin, and I haven't pored over the code enough to figure out why turning it on in the DB isn't working. I'm trying to get Socioboard 4.0 installed instead now, but I haven't given it my full attention yet.

Where are the forums located?

And where are you currently along in the process of trying to get it to work? I feel like my time is better spent helping you and anyone else get to where I am at this point.

**Comment by TechnologyClassroom** [comment]
I have also seen some projects use MariaDB and Redis.

There is a MongoDB replacement in the works called [FerretDB](https://www.ferretdb.io/).  Maybe that would work as a replacement.

**Comment by sureshbabu1995** [comment]
Hi,

For this time, we are using aMember as the Admin panel for managing 
users access, payments, and other activities.

Please follow the link to know about aMember Pro and if you're 
interested to use it, I can help you to configure this and have admin 
Dashboard for you.

aMember Pro Link: http://url1318.globussoft.in/ls/click?upn=wVOYq0Yhcm6rutzZVQWOeLziLhT-2FNulI2p6AeTzkGbMNjyGuG4-2FmnR9sqUZVocxzBWD__U31jt5HgHBh7k3Grc5s4lyrUDWgQ67edTfedIdprlt-2Bj2AMI1oyzLxWnsF19Z4XDbXuT5knH35ZdpCMVaCgrn-2BlohNeQ6NYJDJVI7aW0j5gck1D21kL0HzmVs2D8yKNJQewy2gonmnpnyaikKewFIB-2FXfr8YxSGn6asvO6lhWN7uEk998BQJtstyADDKa9-2BDtfghD0L49bXf8xOVGJ2mHb-2FTGyCBE-2FSUtvl1vdkO5dTRvflp3dZAiVyl93F6RvkbhnKSGjA8R3H8-2FRT23kROOQ-3D-3D

On 3/22/2022 4:00 AM, smokinjo wrote:
>
> HI NIck,
>
> Thanks for the reply.
>
> We have made some progress, but the system is not working. We've made 
> progress by trying to fix errors as they show up. WE still can not log 
> in to the system.
>
> I am not looking forward to seeing how it handles posting messages and 
> monitoring them. Hopefully this part is running better than getting it 
> working to be able to log in.
>
> Have you installed this yourself?
>
> The developers suggest help in the forums, but after trying the 
> forums, email, etc... The developers have not been forthcoming with 
> assistance or words of guidance.
>
> Thanks
>
> —
> Reply to this email directly, view it on GitHub 
> <http://url1318.globussoft.in/ls/click?upn=wVOYq0Yhcm6rutzZVQWOeAyTR0OGTuOgP7Fm7H6pahhay48TmT0HRmOtYb3kDJlO8nXTLgAfkX1KBH6HaTZ7V24tfb-2BF9kp1vdQ2VKVb47dqavKBBBhdMt9eG8aWdVRXZcbK_U31jt5HgHBh7k3Grc5s4lyrUDWgQ67edTfedIdprlt-2Bj2AMI1oyzLxWnsF19Z4XDbXuT5knH35ZdpCMVaCgrn-2BlohNeQ6NYJDJVI7aW0j5gck1D21kL0HzmVs2D8yKNJu-2BvGc853xJDlbMBGsvd1vIHFB-2BE88OgBhymfGvQ5xnzGqMuRKhp8o1-2BUFqieVRuwRVa7dO9TwpqTctHoKY7r-2BzBRsTVsq1VZJbb38ROefBsh-2FK20lq6w1QLBe-2BMN6jtMIJtFqh3d28dgrYZsUKRfAQ-3D-3D>, 
> or unsubscribe 
> <http://url1318.globussoft.in/ls/click?upn=wVOYq0Yhcm6rutzZVQWOeJdJAXOe3xeRHvz9i2Gt-2FidGQ8Csp5o5DNksqKyd-2F4nvXg-2FE5h3-2FiE-2F7cNP3KJW9Sbs-2Fmd072pTPZjavpfXWYvA39x9pdJlrJbHZUbe-2FCZ7Vj-2B2x-2FInDH4OHqgxc73xmlw-3D-3DY21l_U31jt5HgHBh7k3Grc5s4lyrUDWgQ67edTfedIdprlt-2Bj2AMI1oyzLxWnsF19Z4XDbXuT5knH35ZdpCMVaCgrn-2BlohNeQ6NYJDJVI7aW0j5gck1D21kL0HzmVs2D8yKNJDKD0y0ecx2urDBZ6J8CoxRz0EtfDBXc4fqXfdDgRmun8maGvv3KFjEFE4PGyXN6hlKval6HqYQUmKpjwVohIg6SWZE5xFP-2F3t8YmKL0cZO0h0jlJViS9jANdBokGEzKO0bKrD6bzBBBFtuMOMHfSsw-3D-3D>.
> You are receiving this because you are subscribed to this 
> thread.Message ID: 
> ***@***.***>
>

-- 
Thanking You

Best Regards,
Gogula Suresh Babu
Senior Software Engineer(TL)
*
*Bhilai Office:*          First Floor, CHPL Dream Home Apartment,
                               Near Dena Bank, Apollo Hospital Road, 
Junwani,
                               Bhilai, Dist. - Durg, Chhattisgarh, Pin - 
490020

*Bangalore Office :*2nd Floor, Lakshya Towers, KHB Colony, 5th Block
                              Koramangala, Bangalore, Karnataka, Pin - 
560095

Globussoft Global Helpline : 0788-4083007, 0788-4083008
Globussoft Bangalore Helpline : 080-41660003, +91-8550020304
*

**Reply by smokinjo** [reply]
HI there.

Thanks for th eidea.

But, this amember solution seems to be for running a full fledged social media management site with lots of client users.

What about those who are looking to use this as a tool for our 2 teams?

We just want to be able to create a small number of users, and the members of each team would be grouped together n dcould work together on the same accounts.

How can one achieve this?

**Reply by NickAtkinson** [reply]
> For this time, we are using aMember as the Admin panel for managing users access, payments, and other activities.

Thanks for this piece of information, it helps me out a lot. I'm not quite ready to purchase aMember right now, but it's a strong possibility in the not-too-distant future.

I'm also in the process of installing Socioboard 4.0, and I took a quick look at a few config files and didn't see any mention of aMember. Is it safe to assume that there's an admin panel built into Socioboard 4.0?
